core/include/config.h

```diff
@@ -0,0 +1,16 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright 2019 Pengutronix
+ * All rights reserved.
+ *
+ * Rouven Czerwinski <entwicklung@pengutronix.de>
+ */
+#ifndef __CONFIG_H_
+#define __CONFIG_H_
+
+#define __stringify(s) #s
+
+#define IS_CONFIGURED(feat)			\
+(__stringify(feat) == "1")
```

Comment:
How come this works? How can we know that both strings will have the same pointer value?

Reply:
I guess its because they end up being the same value and thus get referenced to the same address. But this is surely compiler specific, I'll go back to the drawing board.